### ISSUE TITLE
Bot was unauthorized to reserve ticket

### ISSUE THREAD

**Issue by SkyOfDoom** [AleksiVirkkala/KideAppBot#5]
when trying to send reservation, this was the response:
"Unauthorized. Get a new token from kide app and try again"

There is nothing wrong with the token because i just put it in and logged into the bot right before this.

![image](https://github.com/AleksiVirkkala/KideAppBot/assets/149057467/0da92a4b-7f81-4624-b2d6-092d7a848466)

**Comment by AleksiVirkkala** [comment]
Kide.app made just yesterday changes to how the bearer token is formed on their side. For example, when signing in on https://kide.app, it now uses cloudflare captcha to verify user. This is why users can no longer sign in to the bot using only their username and password.

It seems that tokens that were created before yesterday don't work anymore (even though they are technically valid), which means users have to sign in to kide.app again and find the new token with web inspector. This worked for me at least. 

Can you test if this method fixes your issue?

**Comment by SkyOfDoom** [comment]
Hi,
I logged out on all devices and copied the code again.
I tried again but unfortunately It still does not work.

The bot seems to work fine but it just cant reserve the tickets, thus the "unauthorized" problem.

EDIT: SOLVED
I got it working again, and the problem was sneaky.

I "triple clicked" on the code before and pasted it directly to the "insert bearer token box". Then i got the unauthorized problem.

However if i copied the code and pasted it in notepad, i noticed that the "" citation marks was also copied. This was the problem for me. I removed the characters and pasted the code back in the bot, and it worked and reserved the tickets.

So if anyone have the same problem make sure you copied the WHOLE code WITHOUT "" marks.
